### PR TITLE
vegeta 5.8.0

### PR DESCRIPTION
**Pull request by tinnet**
just replaced source tar.gz, hash, and version :)

changelog:

>2015-08-11: v5.8.0
>  Change reporters quantile estimation method to match R's 8th type.

https://github.com/tsenart/vegeta/blob/master/CHANGELOG

### CODE REVIEW

File: Library/Formula/vegeta.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def install
     ENV["CGO_ENABLED"] = "0"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "-ldflags", "-X main.Version v5.7.1", "-o", "vegeta"
+    system "go", "build", "-ldflags", "-X main.Version v5.8.0", "-o", "vegeta"
```
Comment (mikemcquaid) [inline Library/Formula/vegeta.rb]: Use "#{version}` in here.

Comment (tinnet) [inline Library/Formula/vegeta.rb]: where does that variable come from? do i have to set it within the formula?

Comment (mikemcquaid) [inline Library/Formula/vegeta.rb]: Nope, set for you by default. Comes from the URL.

**Comment by tinnet** [comment]
done, do you want me to squash the commits or is it ok like this? :)

**Comment by dunn** [comment]
We prefer a single commit in most cases, yeah. I took care if it here, though! Merged in adb7bfb5aca60b1e8c815e3fc0d941616d62ad02, thanks again.

**Pull request closed without merging**